sklearn/metrics/tests/test_classification.py

```diff
@@ -199,16 +199,18 @@ def test_precision_recall_f1_score_binary():
 
 @ignore_warnings
 def test_precision_recall_f_binary_single_class():
-    # Test precision, recall and F1 score behave with a single positive or
+    # Test precision, recall and F-scores behave with a single positive or
     # negative class
     # Such a case may occur with non-stratified cross-validation
     assert 1. == precision_score([1, 1], [1, 1])
     assert 1. == recall_score([1, 1], [1, 1])
     assert 1. == f1_score([1, 1], [1, 1])
+    assert 1. == fbeta_score([1, 1], [1, 1], 0)
 
     assert 0. == precision_score([-1, -1], [-1, -1])
     assert 0. == recall_score([-1, -1], [-1, -1])
     assert 0. == f1_score([-1, -1], [-1, -1])
+    assert 0. == fbeta_score([-1, -1], [-1, -1], float('inf'))
```

Comment:
Also please add a continuity test there,
```
assert fbeta_score([-1, -1], [-1, -1], float('inf')) == pytest.approx(
   fbeta_score([-1, -1], [-1, -1], beta=1e5)
```
possibly with the appropriate tolerance.